scripts/Units/armcroc.bos

```diff
@@ -130,6 +148,74 @@ AimFromPrimary(piecenum)
 	piecenum = turret;
 }
 
+
+
+
+
+
+
+
+
+AimSecondary(heading, pitch)
+{
+	signal SIG_AIM2;
+	set-signal-mask SIG_AIM2;
+	
+	move aturret to y-axis [15] speed [100];
+	move launcher to y-axis [4.5] speed [100];
+	turn launcher to x-axis <90.000000> speed <200.000000>;
+	
+	if( gun_2 == 1 )
+	{
+		turn asleeve to y-axis <0.000000> speed <400.000000>;
+	}
+	if( gun_2 == 2 )
+	{
+		turn asleeve to y-axis <120.027473> speed <400.000000>;
```

Comment:
`120.027473` looks oddly precise

Reply:
yup, so I'm leaving it as was in case there was actually some weird edge case reason for that